React-Native/en/3.1.2.md

```diff
@@ -0,0 +1,7 @@
+# [Success Criterion 3.1.2 - Language of Parts](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html)
+
+Unfortunately, React Native only has support for changing the language of parts of the content on iOS. You can do this for iOS by setting the [`accessibilityLanguage`](https://reactnative.dev/docs/text#accessibilitylanguage-ios).
```

Comment:
add prop suffix behind accessibilityLanguage?

Maybe link to android 3.1.2 for the native solution?

Reply:
Good one, I added a link to Android 3.1.2 for the native solution. I don't really understand what you mean by "add prop suffix behind accessibilityLanguage?". There's a code example underneath to show how to set a language, do you want to add that in text too?

Reply:
Ah, I meant:

```
You can do this for iOS by setting the [`accessibilityLanguage`](https://reactnative.dev/docs/text#accessibilitylanguage-ios) prop.
```

e.g. placing 'prop' behind the accessibilityLanguage word